data_structures/queue_using_one_stack.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+  @author [Rajat Garg](https://github.com/Rajat379)
+  @file
+
+  Implementation of a Queue using one Stack.
+*/
+
+#include <cassert>
+#include <iostream>
+#include <stack>
+
+namespace {
+/**
+ * @brief Queue data structure. Stores elements in FIFO
+ * (first-in-first-out) manner.
+ * @tparam T datatype to store in the queue
+ */
+template <typename T>
+class QueueWithStack {
+ private:
+    std::stack<T> stackQueue;
```

Comment:
Please add a one-line description of what the variable is for (**see the example below**).

```c++
int64_t data = 0;       ///< The value/key of the node.
```